docs/content/docs/tutorials/ec2-example.md

```diff
@@ -0,0 +1,477 @@
+  Owner ID:                         515336597380
+  Private DNS Name Options On Launch:
+  State:      available
+  Subnet ID:  subnet-<ID>
+ 
+```
+ 
+### Validate
+ 
+This network setup should allow Instances deployed in the Private Subnet to connect to the internet. To validate this behavior deploy an Instance into the Private Subnet and the Public Subnet (bastion host). After deployments, `ssh` into the bastion host, then `ssh` into the Private Subnet Instance, and test internet connection. Security group is required by both instances launched in public and private subnets. 
+
+Note, we need to provide Subnet and SecurityGroup ID's in the yaml manually; run `kubectl describe subnets` and `kubectl describe securitygroups` commands to get ID's.
+ 
+* Deploy an Instance into the Private Subnet using provided yaml and `kubectl apply -f instance-private-subnet.yaml`:
+ 
+```
+apiVersion: ec2.services.k8s.aws/v1alpha1
+kind: Instance
+metadata:
+  name: tutorial-instance-private
+spec:
+  imageID: ami-02b92c281a4d3dc79 # AL2; us-west-2
+  instanceType: c3.large
+  subnetID: subnet-<private-ID>
+  securityGroupIDs:
+  - sg-<ID>
+  keyName: us-west-2-key # created via console
+  tags:
+    - key: producer
+      value: ack
+```
+ 
+* Deploy an Instance into the Public Subnet using provided yaml and `kubectl apply -f instance-public-subnet`:
+ 
+```
+apiVersion: ec2.services.k8s.aws/v1alpha1
+kind: Instance
+metadata:
+  name: tutorial-instance-public
+spec:
+  imageID: ami-02b92c281a4d3dc79 # AL2 in us-west-2
+  instanceType: c3.large
+  subnetID: subnet-<public-ID>
+  securityGroupIDs:
+  - sg-<ID>
+  keyName: us-west-2-key # created via console
+  tags:
+    - key: producer
+      value: ack
+```
+ 
+* Deployed 2 instances; one to each Subnet
+  * The instance in the public subnet will be the bastion host so we can ssh to the Instance in the private Subnet
+    ```bash
+    scp "/path/created_key_in_console_for_region.pem" ec2-user@<Public IPV4 DNS>:
+    ssh -i "/path/created_key_in_console_for_region.pem" ec2-user@<Public IPV4 DNS>
+    ssh -i "created_key_in_console_for_region.pem" ec2-user@<Private IP>
+    ```
+* Validate instance in private subnet can connect to internet
+  * Try to ping websites from your private subnet, sample output looks like
+    ```bash
+    ping google.com
+  
+    PING google.com (142.250.217.78) 56(84) bytes of data.
+    64 bytes from sea09s29-in-f14.1e100.net (142.250.217.78): icmp_seq=1 ttl=102 time=8.30 ms
+    64 bytes from sea09s29-in-f14.1e100.net (142.250.217.78): icmp_seq=2 ttl=102 time=7.82 ms
+    64 bytes from sea09s29-in-f14.1e100.net (142.250.217.78): icmp_seq=3 ttl=102 time=7.77 ms
+    ^C
+    --- google.com ping statistics ---
+    3 packets transmitted, 3 received, 0% packet loss, time 2003ms
+    ```
+ 
+### Cleanup
+ 
+Remove all the resources using `kubectl delete` command.
+
+```bash
+kubectl delete -f tutorial-instance-public.yaml
+kubectl delete -f tutorial-instance-private.yaml
```

Comment:
These files had different names earlier in the tutorial. Recommend using the same names, whichever one you pick.